app/views/pages/sponsor.html.haml

```diff
@@ -0,0 +1,107 @@
+= provide :title, 'Sponsor'
+
+.introduction.main-section
+  .row
+    %span.vertical-align
+      %h1
+        <b><span class="logo-h">h</span>ackEDU</b> empowers high school
```

Comment:
Perhaps make a partial for `<b><span class="logo-h">h</span>ackEDU</b>` if we're going to keep using this. Also, `<b>` is a style and therefore should be in CSS.